pages/home.html

```diff
@@ -100,10 +106,8 @@
     Contributing to Oppia
   </h2>
   <p>
-    Oppia is built by the community for the community. Our mission is to enable
-    anyone to learn anything they want to in an effective, enjoyable way, and
-    building a platform that allows anyone to create and share bite-sized
-    interactive learning material is just a first step.
+    <em>Built by the community for the community</em> – Oppia aims to enable
```

Comment:
I want to insist on this spirit, I think the expression _Built by the community for the community_ is like a motto, a principal property of Oppia. It is too long to be a headline though. It can create a strong impression on readers when they see this motto first, and read the next content later.